plugin/src/main/scala/org/flywaydb/play/ConfigReader.scala

```diff
@@ -66,6 +66,8 @@ class ConfigReader(app: Application) extends UrlParser {
         app.configuration.getBoolean(s"db.${dbName}.migration.outOfOrder").getOrElse(false)
       val auto =
         app.configuration.getBoolean(s"db.${dbName}.migration.auto").getOrElse(false)
+      val cleanOnStart =
+        app.configuration.getBoolean(s"db.${dbName}.migration.cleanOnStart").getOrElse(false)
```

Comment:
@vjkoskela i want to rename cleanOnStart to cleanOnStartForTest to clarify this option is only for testing and never works in production environment. what do you think?